### PR TITLE
Revert infinite loop bug

### PR DESCRIPTION
**Pull request by EricPKerr**
This reverts a critical error described here: https://github.com/Marak/colors.js/commit/074a0f8ed0c31c35d13d28632bd8a049ff136fb6

### CODE REVIEW

**Review by dfederm — APPROVED** [review]

**Review by mikolajkalwa — APPROVED** [review]

**Review by wiktorkowalski — APPROVED** [review]

**Review by nooreldeenmansour — APPROVED** [review]

File: package.json
Comment (meiram-tr) [inline package.json:4]: better use 1.4.5 in order to updated all

**Review by Solixity — CHANGES_REQUESTED** [review]
Check `colors/safe` as well, 1.4.2 also has the same issue.

**Review by brahian-pena — APPROVED** [review]

File: package.json
Comment (EricPKerr) [reply package.json:4]: The problematic previous version is 1.4.44 (-liberty-2).  So it would have to be 1.4.45 if you think it needs to be a minor version.  Was just attempting to make it simple for people to avoid all of the 1.4.X code given the severity of this.

**Comment by oliviergf** [comment]
LGTM :heavy_check_mark:

**Review by deyvisonrocha — APPROVED** [review]

**Comment by EricPKerr** [comment]
I'm unable to merge or npm publish this.  Is a maintainer able to do that?

**Comment by Nivl** [comment]
It won't be merged. The author did this on purpose, it's not a mistake nor a bug. They did similar things on their other packages, such as Faker.js (they removed the source, and push v6.6.6 to NPM that breaks everything).

**Review by Kowalski-Mateusz — APPROVED** [review]

**Comment by G2G2G2G** [comment]
Don't merge this, it breaks functionality of the new library and how I use it.

**Review by hello-smile6 — APPROVED** [review]
Stop breaking people's code!!! If you want, log a polite message, but dOS is not acceptable!

**Review by TechStudent10 — APPROVED** [review]

**Review by hello-smile6 — APPROVED** [review]

**Review by stifskere — APPROVED** [review]

File: lib/index.js
Comment (stifskere) [inline lib/index.js:17]: Just, do that in one line

```js
require('../lib/custom/american')();
```

**Review by PululuK — APPROVED** [review]

**Comment by tj-commits** [comment]
@G2G2G2G I whole-heartedly agree. except npm unpublished 1.44.4-liberty-2

**Comment by tj-commits** [comment]
> Don't merge this, it breaks functionality of the new library and how I use it.

Let's go! you're right

**Pull request closed without merging**
